maxmind.json

```diff
@@ -338,6 +338,118 @@
                         "failed"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.Date",
```

Comment:
Data path here are copied paste from:
![image](https://user-images.githubusercontent.com/82124762/147983226-0b6ef84b-144d-4502-b39c-99a708b64cd9.png)

Reply:
This should be more clear in the README as well:
![image](https://user-images.githubusercontent.com/82124762/147986113-bc64b464-6cfb-4f76-867f-332beeb54426.png)